resources/config-defaults.edn

```diff
@@ -14,13 +14,19 @@
  ;; Path to a writable directory where to store the Lucene search index files.
  :search-index-path "search-index"
 
- ;; Supported authentication methods: :shibboleth, :fake-shibboleth, :ldap
+ ;; Supported authentication methods: :shibboleth, :fake-shibboleth, :ldap, :oidc
  :authentication :shibboleth
  :ldap {:connection {:host "localhost:2636"
                      :ssl? true}
         :search-root "dc=Suivohtor,dc=local"}
  :alternative-login-url nil
 
+ ;; Open ID Connect settings
+ ; These allow login to http://localhost:3000/oidc-callback only
+ :oidc-domain "remsdev.eu.auth0.com"
+ :oidc-client-id "65TIN1Joo6EvsHSPCN4zyCaxxPh0WJF1"
+ :oidc-client-secret "e1yr9R_8WW3Xg03wm6wJ4UmNVVjiqdVhwFGvHhttyoRS3h04F4MGh0zRaV7vKm30"
```

Comment:
These should be in `dev-config.edn` and not defaults. Otherwise it's possible to login to a production system which accidentally doesn't override the default values.

Reply:
Sure, can move them. But the IdP never allows callbacks to unconfigured domains, so there's a double check.